app/assets/javascripts/map.js

```diff
@@ -52,28 +52,16 @@ class Map {
     })
   }
   
-  initRemoveLastControl() {
-    const removeLastControlDiv = document.createElement('button');
-    removeLastControlDiv.classList.add('map-btn')
-    removeLastControlDiv.title = 'Click to remove selected area from the map';
-    removeLastControlDiv.innerHTML = 'Remove last';
+  initRemoveLastControls(title, label) {
```

Comment:
Nothing in this method makes it specific to "Remove Last", what makes these "remove" specific, is just the labels you are attaching to them".  It's just a generic `initButtonControl`